runelite-client/src/main/java/net/runelite/client/plugins/experiencedrop/XpDropConfig.java

```diff
@@ -87,4 +87,14 @@ default int fakeXpDropDelay()
 		return 0;
 	}
 
+	@ConfigItem(
+		keyName = "disableFakeXpDrops",
+		name = "Disable fake xp drops",
+		description = "Enable to disable post 200m fake exp drops.",
```

Comment:
```java
description = "Disables post-200m fake exp drops",
```

Reply:
Don't ironmen also get this when attacking other players in the wilderness? Or did that not get implemented?

Reply:
Also, you changed the wrong description

Reply:
Ironmen do receive these when attacking other players